src/Event/Commands/CreateEvent.php

```diff
@@ -122,11 +122,12 @@ public function getLocation()
         return $this->location;
     }
 
-    /**
-     * @return DateTimeImmutable|null
-     */
-    public function getPublicationDate()
+    public function getPublicationDate(DateTimeImmutable $default): ?DateTimeImmutable
```

Comment:
Is there still a use case to make the return type nullable? If the `$this->publicationDate` is `null`, can't we also return the `$default`? 🤔

Reply:
I don't think so, because I think it's possible to create events without publication date and then publish them manually later.

Reply:
Ah yes I overlooked the concept of drafts :)